File: src/main/java/moe/tristan/kitsumonogatari/KitsumonogatariApplication.java
package moe.tristan.kitsumonogatari;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;

import moe.tristan.easyfxml.EasyFxmlAutoConfiguration;
import moe.tristan.easyfxml.FxApplication;


@SpringBootApplication
@Import(EasyFxmlAutoConfiguration.class)
public class KitsumonogatariApplication extends FxApplication {

    public static void main(String[] args) {
        launch(args);
    }

}

File: src/main/kotlin/moe/tristan/kitsumonogatari/KitsumonogatariApplication.kt
package moe.tristan.kitsumonogatari

import moe.tristan.easyfxml.spring.application.FxSpringApplication
import moe.tristan.easyfxml.spring.application.FxSpringContext
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.context.annotation.Import
import java.awt.Toolkit

@SpringBootApplication
@EnableCaching
@Import(FxSpringContext::class)
class KitsumonogatariApplication: FxSpringApplication() {

    companion object {
        fun main(args: Array<String>) {
            Toolkit.getDefaultToolkit()
            launch(KitsumonogatariApplication::class.java)
        }
    }

}
